lib/weapons.json

```
@@ -2944,6 +2944,58 @@
       "Melee": 1
     }
   },
+  {
+    "id": "coherentsubatomicneedle",
+    "name": "Coherent Subatomic Needle",
+    "mount": "Main",
+    "type": "Melee",
+    "damage": [
+      {
+        "type": "kinetic",
+        "val": "1d6"
+      }
+    ],
+    "range": [
+      {
+        "type": "Threat",
+        "val": 2
+      }
+    ],
+    "tags": [],
+    "source": "SSC",
+    "license": "WHITE WITCH",
+    "license_level": 1,
+    "effect": "1/round you may force a character struck by this weapon must make a hull save. On a failed save, both you and the target are immobilized. At the end of each of its turns, the target can repeat the save, freeing both of you on success. As a quick action on your turn, you can break this immobilization effect on both of you to whip your target away, knocking them back 4 spaces in any direction. Otherwise you both remain immobilized.",
```

```suggestion
    "effect": "1/round you may force a character struck by this weapon to make a hull save. On a failed save, both you and the target are immobilized. At the end of each of its turns, the target can repeat the save, freeing both of you on success. As a quick action on your turn, you can break this immobilization effect on both of you to whip your target away, knocking them back 4 spaces in any direction. Otherwise you both remain immobilized.",
```
Not entirely sure about whether this fix was intended, but it sounds better...